packages/react/src/themes/teams/components/Icon/iconStyles.ts

```diff
@@ -1,14 +1,14 @@
 import * as _ from 'lodash'
 
 import fontAwesomeIcons from './fontAwesomeIconStyles'
-import { callable, pxToRem } from '../../../../lib'
+import { callable, pxToRem, SizeValue } from '../../../../lib'
 import { ComponentSlotStylesInput, ICSSInJSStyle, FontIconSpec } from '../../../types'
 import { ResultOf } from '../../../../types'
-import { IconXSpacing, IconProps, IconSize } from '../../../../components/Icon/Icon'
+import { IconXSpacing, IconProps } from '../../../../components/Icon/Icon'
 import { getStyle as getSvgStyle } from './svg'
 import { IconVariables, IconSizeModifier } from './iconVariables'
 
-const sizes: { [key in IconSize]: number } = {
+const sizes: { [key in SizeValue]: number } = {
```

Comment:
```suggestion
const sizes: Record<SizeValue, number> = {
```

As I remember this should work